dvc/config.py

```diff
@@ -149,6 +149,8 @@ class RelPath(str):
                     "profile": str,
                     "credentialpath": str,
                     "endpointurl": str,
+                    "s3_key_id": str,
+                    "s3_key_secret": str,
```

Comment:
Kinda have an urge to just do
```suggestion
                    "endpointurl": str,
                    "key_id": str,
                    "key_secret": str,
```
or, even better:
```suggestion
                    "endpointurl": str,
                    "access_key_id": str,
                    "secret_access_key": str,
```
to make it easier to find and more intuitive in general. Considering that `oss` has
```
                    "oss_key_id": str,
                    "oss_key_secret": str,
                    "oss_endpoint": str,
```
already (which we could consider unifying in the future).